### PR TITLE
[CI][License]Exclude license check of example

### PR DESCRIPTION
**Pull request by CalvinKirs**
because it won't be in our binary tar

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: tools/dependencies/checkLicense.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 
-./mvnw dependency:copy-dependencies -DincludeScope=runtime -DoutputDirectory=/tmp/seatunnel-dependencies
+./mvnw -pl '!seatunnel-examples,!seatunnel-examples/seatunnel-spark-examples,!seatunnel-examples/seatunnel-flink-examples' dependency:copy-dependencies -DincludeScope=runtime -DoutputDirectory=/tmp/seatunnel-dependencies
```
Comment (kezhenxu94) [inline tools/dependencies/checkLicense.sh]: can you try to add a plugin declaration in `seatunnel-examples` with `skip=true`, so when people add new examples they don't need to modify this file.

in file `seatunnel/seatunnel-examples/pom.xml`, the plugin should be extended by all children modules

```xml
<build>
  <plugins>
    <plugin>
      <groupId>org.apache.maven.plugins</groupId>
      <artifactId>maven-dependency-plugin</artifactId>
      <configuration>
        <skip>true</skip>
      </configuration>
    </plugin>
....
```

Comment (xleoken) [reply tools/dependencies/checkLicense.sh]: Good idea.

Comment (CalvinKirs) [reply tools/dependencies/checkLicense.sh]: Good idea.thx:)

**Review by xleoken — APPROVED** [review]
👍

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**